pentaho-webpackage/extenders/http-extender/src/test/java/org/pentaho/webpackage/extender/http/impl/PentahoWebPackageServiceImplTest.java

```diff
@@ -144,7 +150,119 @@ public void addWebPackageBundle() {
 
     assertResourceMappingExists( capturedResourceMappings, "/pentaho-webpackage-1a", "/package-name-1a/1.0" );
     assertResourceMappingExists( capturedResourceMappings, "/pentaho-webpackage-1b", "/package-name-1b/1.1" );
-    assertResourceMappingExists( capturedResourceMappings, "/pentaho-webpackage-1c", "/package-name-1c/1.2" );
+    assertResourceMappingExists( capturedResourceMappings, "/pentaho-webpackage-1c", "/@organization/package-name-1c/1.2" );
```

Comment:
Didn't we settle on an initial `/` being an invalid AMD id? Having this be used in tests for normal use cases doesn't provide much confidence...

Reply:
@dcleao this is related with the expected alias for the DefaultResourceMapping .. the actual check for the initial `/` was removed from the code like we settled.